ballerina/Ballerina.toml

```diff
@@ -2,12 +2,12 @@
 distribution = "2201.8.0"
```

Comment:
Better to bump the distribution version to `2201.10.0` as this is a possible breaking change. If we are doing that need to update the Ballerina lang version in `gradle.properties` as well.

Reply:
Updated!